.github/workflow/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Run tests
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+jobs:
+    test:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
Can you please keep 2.7 for this particular repository, for now? We normally don't go out of our way to support EOL python versions, but this particular project still runs in a number of older places and doesn't change often.

We might actually have to drop the support when we merge your other PR because the new toolchest dependency only supports Python 3, but for now let's keep things as they were: this is only about adding CI.